src/util/getArticleScripts.ts

```diff
@@ -6,9 +6,19 @@
  *
  */
 
-export function getArticleScripts(article) {
-  const scripts =
-    article?.requiredLibraries.map(lib => ({
+import { GQLArticle } from '../graphqlTypes';
+
+interface Scripts {
+  key?: string;
+  src?: string;
+  type?: string;
+  async?: boolean;
+  defer?: boolean;
+}
+
+export function getArticleScripts(article: GQLArticle) {
```

Comment:
Er `article` nullable? Det sjekkes for i linje 25. Kunne også vært greit å flytte de statiske scriptene ut av funksjonen, slik at den ble enklere å lese.

Reply:
Den er nullable, og flytte statiske scriptene er satt til statiske konstanter utenfor